File: week-03/day-3/06.py
# create a function that takes a list and returns a new list that is reversed
def list_reverse(list1):
    result = []
    for i in range(len(list1)-1, -1, -1):
        result = result + [list1[i]]
    return result

print(list_reverse([1,2,3,4,5,6]))

File: week-07/day-03/06.js
'use strict';


// create a function that takes a string and a letter and returns a boolean
// it should return true if the string consits the given letter, false otherwise

function isInString(inputString, inputLetter) {

   return inputString.split('').some(function(e){
      return e === inputLetter;
   })

}

console.log(isInString('kutya', 'a'));
console.log(isInString('kutya', 'z'));
